lib/matomo.dart

```diff
@@ -345,9 +345,12 @@ class _Event {
         map['_rck'] = this.tracker.campaign!.keyword;
         entries.addEntries([MapEntry('utm_term', this.tracker.campaign!.keyword!)]);
       }
-      if (this.tracker.campaign!.gclid != null) {
-        map['gclid'] = this.tracker.campaign!.gclid;
-        entries.addEntries([MapEntry('gclid', this.tracker.campaign!.gclid!)]);
+      if (this.tracker.campaign!.identifiers != null) {
+        final identifiers = tracker.campaign!.identifiers;
+        identifiers?.forEach((key, value) {
+          map[key.toLowerCase()] = value;
```

Comment:
Why the `.toLowerCase()`?

Reply:
That was my suggestion, but I had a look at the http protocol and parameters should be case sensitive and this wouldn't follow the protocol, so I'll fix it